applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -218,8 +227,11 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	size_t num_streams = ARRAY_SIZE(streams);
 	size_t data_size = size / num_streams;
 
-	if (streams[0].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
-		return -ECANCELED;
+	for (size_t i = 0U; i < num_streams; i++) {
```

Comment:
Couldn't you here check the all_streams_streaming instead of checking again, and then in stream_stopped_cb set all_streams_streaming to false?

Edit: I saw that all_streams_streaming isn't global, it could be though...

Reply:
Moving this check to the for-loop below